.github/workflows/publish-crates.yml

```diff
@@ -27,6 +27,9 @@ permissions:
 jobs:
   publish_crates:
     runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash
```

Comment:
Let's make this global across all jobs. I think placing this right after https://github.com/build-trust/ockam/blob/cdf925aa2adb4439061449e5ea5f9ad774833e20/.github/workflows/publish-crates.yml#L23-L25 sounds okay.

Reply:
@metaclips please check now

Reply:
Nice! Thanks!!